geth/txqueue/txqueue_manager_test.go

```diff
@@ -124,36 +156,45 @@ func (s *TxQueueTestSuite) TestCompleteTransactionMultipleTimes() {
 
 	txQueueManager.SetTransactionReturnHandler(func(queuedTx *common.QueuedTx, err error) {
 		s.Equal(tx.ID, queuedTx.ID)
-		s.Equal(errTxAssumedSent, err)
+		s.NoError(err)
 	})
 
 	err := txQueueManager.QueueTransaction(tx)
 	s.NoError(err)
 
 	var wg sync.WaitGroup
 	var mu sync.Mutex
-	completeTxErrors := make(map[error]int)
-	for i := 0; i < 3; i++ {
+	var completedTx int
```

Comment:
Minor: as there are multiple `var`s, it's beneficial to group them with `var ()`.